packages/client/src/comp/states/Minnesota.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+
+import Select from 'muicss/lib/react/select'
+import Option from 'muicss/lib/react/option'
+
+import { MinnesotaInfo, minnesotaIdentityType, isMinnesotaIdentity} from '../../common'
+import { SignatureBase, StatelessInfo, NoSignature } from './Base'
+import { useControlRef } from '../util/ControlRef'
+import { BaseInput } from '../util/Input'
+
+export const Minnesota = () => {
+  const idTypeRef = useControlRef<Select>()
+  const idDataRef = useControlRef<Select>()
+
+  const enrichValues = (baseInfo: StatelessInfo): NoSignature<MinnesotaInfo> | null => {
+    const idType  = idTypeRef.value()
+    if (!isMinnesotaIdentity(idType)) return null
+
+    const idData = idDataRef.value()
+    if (!idData) return null
+
+    return {
+      ...baseInfo,
+      state: 'Minnesota',
+      idType,
+      idData,
+    }
+  }
+
+  return <SignatureBase<MinnesotaInfo>enrichValues={enrichValues}>
+    <p>Minnesota requires voters to confirm their identify using one of the following types of identification</p>
+    <Select ref={idTypeRef} label='Identification Type' defaultValue='Select' {...{required: true}}>
+      <Option key={0} hidden={true}/>
+      {[...minnesotaIdentityType].sort().map((value, key) => {
+        return <Option value={value} key={key+1} label={value}/>
+      })}
+    </Select>
+    <p>Enter the relevant information based on your choice above.  If &apos;None&apos; 
```

Comment:
Note this.  To be honest, I did it because I don't know how to make the input component toggle-able based on the value of the drop down (should be hidden if user chooses 'None'), but this actually may be a useful way to force the user to acknowledge the fact that they are telling the election official that they do not have a form of idenitification.